### PR TITLE
Expand buttons and other cleanup

### LINKED ISSUE #8: Improve dropdown buttons
Dropdown buttons currently lack
- toggled styling
- good accessibility labels
- better titles

### PR DESCRIPTION
**Pull request by tadhgj**
index.html:
Changed title, meta tags
Removed old analytics
Added <span> for expand-button label

application.js:
Reworked expand-button logic
- scrolling
- add active class

css:
fixed unaligned "tweet" button on post-game screen
expand-button now has
- no width limit to fit text
- better font and color
- active class to indicate selected section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,28 +3,29 @@
...
+    //   }, i[r].l = 1 * new Date(); a = s.createElement(o),
+    //     m = s.getElementsByTagName(o)[0]; a.async = 1; a.src = g; m.parentNode.insertBefore(a, m)
+    // })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
 
-    ga('create', 'UA-51255017-1', 'ronzil.github.io');
-    ga('send', 'pageview');
-    var i = new Image().src = "http://bob.quaji.com/ping.php?d=2048&u=" + escape(document.location) + "&r=" + escape(document.referrer);
+    // ga('create', 'UA-51255017-1', 'ronzil.github.io');
+    // ga('send', 'pageview');
+    // var i = new Image().src = "http://bob.quaji.com/ping.php?d=2048&u=" + escape(document.location) + "&r=" + escape(document.referrer);
```
Comment (maxmmyron) [inline index.html:28]: This stuff can just be removed lol

File: index.html [hunk 2]
```diff
@@ -85,9 +86,9 @@ <h1 class="title">2048 - AI</h1>
       </div>
     </div>
     <div class="expand-button-container">
-      <button class="expand-button" id="tutorial-button" alt="toggle tutorial">🤔</button>
-      <button class="expand-button" id="explanation-button" alt="toggle explanation">🧠</button>
-      <button class="expand-button" id="about-button" alt="toggle credits">👨‍💻</button>
+      <button class="expand-button" id="tutorial-button" alt="toggle tutorial"><span class="btn-label">Tutorial</span>🤔</button>
+      <button class="expand-button" id="explanation-button" alt="toggle explanation"><span class="btn-label">Explanation</span>🧠</button>
+      <button class="expand-button" id="about-button" alt="toggle credits"><span class="btn-label">About</span>👨‍💻</button>
     </div>
```
Comment (maxmmyron) [inline index.html:92]: Sweet, thanks for the improved button labels

would you be able to remove the button alt tags? i never realized it, but these aren't parsed by assistive technology

File: js/application.js [hunk 3]
```diff
@@ -16,27 +16,75 @@ const setDetailContainerHeight = (el, isOpen = false) => {
...
 
+  // scroll as element is opening
+  setTimeout(function() {
+    document.getElementById(id).scrollIntoView({
+      behavior: "smooth",
+      block: "start",
+      inline: "nearest"
+    });
+  }
+  , 400);
```
Comment (maxmmyron) [inline js/application.js:57]: This is nice 👍

File: js/application.js [hunk 4]
```diff
@@ -16,27 +16,75 @@ const setDetailContainerHeight = (el, isOpen = false) => {
...
+    // if el.id != id, close it
+    setDetailContainerHeight(el, el.id == id);
+  });
 
+  // scroll as element is opening
+  setTimeout(function() {
+    document.getElementById(id).scrollIntoView({
+      behavior: "smooth",
+      block: "start",
+      inline: "nearest"
```
Comment (maxmmyron) [inline js/application.js:54]: The `block:"start"` and `inline:"nearest"` properties can be removed as those are the default values for `scrollIntoView`

File: style/main.scss [hunk 5]
```diff
@@ -584,22 +587,36 @@ hr {
...
   transition: 0.2s;
-}
+  // light grey
+  color: #444;
+  font-family: "Clear Sans", "Helvetica Neue", Arial, sans-serif;
 
-button.expand-button:hover {
-  background: rgba(0 0 0 / 10%);
-}
+  // effects
```
Comment (maxmmyron) [inline style/main.scss:602]: Lol I wasn't thinking when writing these styles. Thank you for taking advantage of SCSS unlike me

**Review by maxmmyron — CHANGES_REQUESTED** [review]
Some nice QoL features overall, thank you. Just a few minor changes

The `scrollInt` seems a bit flaky; I got some issues with page scrollY lurching around while scrolling and clicking. Might need revisiting but that's fine for now.

File: index.html (hunk 1 above)
Comment (tadhgj) [reply index.html:28]: done

File: index.html (hunk 2 above)
Comment (tadhgj) [reply index.html:92]: removed alt tags

File: js/application.js (hunk 4 above)
Comment (tadhgj) [reply js/application.js:54]: done

File: style/main.scss (hunk 5 above)
Comment (tadhgj) [reply style/main.scss:602]: common scss W

**Pull request merged**